docs/meshstack.how-to.delete-meshObjects.md

```diff
@@ -0,0 +1,50 @@
+---
+id: meshstack.how-to.delete-meshObjects
+title: How to delete meshObjects in the meshPanel
+---
+
+Part of the cloud lifecyle inside meshStack is the deletion of the meshCustomers, meshProject and meshTenants.
+The following guide provides insight for users and admins how to proceed the deletion of their cloud accounts and meshObjects via the meshPanel
+
+## Step-by-Step Guide
+
+## Delete meshTenant from my meshProject
```

Comment:
please add what rights are necessary to delete the tenant

Reply:
added into each section which role is required